app/src/main/java/com/github/se/project/model/profile/ProfilesRepositoryFirestore.kt

```diff
@@ -74,10 +79,13 @@ class ProfilesRepositoryFirestore(private val db: FirebaseFirestore) : ProfilesR
       onSuccess: () -> Unit,
       onFailure: (Exception) -> Unit
   ) {
+    println("we also here")
```

Comment:
Don't know what you mean here, moreover why not use a Log.e to confirm the task success or fail ?

Reply:
Oh, that is a debugging print I used to see where the error was, which I left in by mistake. Will remove.